bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BufferedChannel.java

```diff
@@ -101,6 +101,11 @@ public synchronized void close() throws IOException {
         if (closed) {
             return;
         }
+
+        super.close();
+        writeBufferStartPosition.set(0);
+        writeBuffer.clear();
```

Comment:
The writeBuffer will be released, do we need to clear it?

Reply:
Agree no need to clear buffer, clear statement removed.